### DISCUSSION TITLE
More visual summary view?

### DISCUSSION THREAD

**Discussion by olemp** [Puzzlepart/did#891] [Ideas]
Should we keep the summary view as is?

![image](https://user-images.githubusercontent.com/7606007/111471612-5d150680-8729-11eb-8a49-bd2113f1667a.png)

.. or can the [Timely](https://memory.ai/timely) style be more "readable"?

![image](https://user-images.githubusercontent.com/7606007/111471691-728a3080-8729-11eb-9672-cc864ca3d6f5.png)

Thoughts @tarjeieo @okms @damsleth?

### Adjustments done after discussion was created
User photo sync: https://github.com/Puzzlepart/did/pull/899
Misc adjustments: https://github.com/Puzzlepart/did/commit/8ced82ad255cbf3812558e3b19c211ec3de62ae5

**Comment by damsleth** [comment]
**Short version: I love almost everything about the timely UI.**
Longer version: There are several things we should consider blatantly stealing from the above UI;

1. From/To dates instead of week numbers in the column headers (could possibly be a user or global setting). People remember dates, not week numbers
2. User avatars: Nice to have, and probably pretty easy to do once we've cracked the MSGraph -> GraphQL -> ArrayBuffer -> DataUri/png song and dance with currentUser avatars
3. Visualising Overtime - useful. Color coding is a nice touch
4. Visualising "missing hours" - we touched in on this in #781 - managers use it to check for unconfirmed weeks
5. Visualising Capacity - useful in instances where some employees are not full time emplyees, or on long time partial sick leave
6. LARGER AND BOLDER FONTS

**Reply by olemp** [reply]
> **Short version: I love almost everything about the timely UI.**
> Longer version: There are several things we should consider blatantly stealing from the above UI;
> 
> 1. From/To dates instead of week numbers in the column headers (could possibly be a user or global setting). People remember dates, not week numbers
> 2. User avatars: Nice to have, and probably pretty easy to do once we've cracked the MSGraph -> GraphQL -> ArrayBuffer -> DataUri/png song and dance with currentUser avatars
> 3. Visualising Overtime - useful. Color coding is a nice touch
> 4. Visualising "missing hours" - we touched in on this in #781 - managers use it to check for unconfirmed weeks
> 5. Visualising Capacity - useful in instances where some employees are not full time emplyees, or on long time partial sick leave
> 6. LARGER AND BOLDER FONTS

1. Agreed. We can add a subscription settings for it **IF** we think someone prefers week numbers? Can you create a issue for this particular one?
2. That could help a lot indeed. For now we could use the [Persona](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona) component? For v0.10.0.  
      *EDIT*: Added https://github.com/Puzzlepart/did/commit/8ced82ad255cbf3812558e3b19c211ec3de62ae5
3. I like the color codes that Timely has. Probably needs more detailing before creating an issue. How do we decide the color scheme? Subscription setting?
4. It's handled now with blank cell. It might be too colorful if we color-code that aswell?
5. .. and remove the postfix `t`?
6. Hmmm.... 🚀

**Reply by olemp** [reply]
Using [Persona](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona) component for now.

![image](https://user-images.githubusercontent.com/7606007/111655361-e13ebb00-8809-11eb-818d-a1b9f6266f2d.png)

Ref https://github.com/Puzzlepart/did/commit/8ced82ad255cbf3812558e3b19c211ec3de62ae5

**Reply by damsleth** [reply]
This is a _massive_ improvement already! 👏

**Reply by olemp** [reply]
@damsleth I like the combo! Thoughts?

![image](https://user-images.githubusercontent.com/7606007/111746364-3de3b980-888e-11eb-9125-c5b60017581c.png)

**Reply by olemp** [reply]
@damsleth Also added user photo sync in https://github.com/Puzzlepart/did/pull/899.
